### PR TITLE
LPD-41498 Change the external reference code in the Kaleo Definition post to avoid duplication error

### PR DESCRIPTION
**Pull request by igor-franca**
Related [Ticket](https://liferay.atlassian.net/browse/LPD-41498)

I am proposing to add playwright tests to the ci:test:workflow routine

### CODE REVIEW

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:workflow

File: modules/test/playwright/tests/portal-workflow-kaleo-designer-web/utils/postSingleApproverCopy.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		await apiHelpers.headlessAdminWorkflow.postWorkflowDefinitionSave(
+			'Copy of Single Approver' + getRandomInt(),
+			{
+				...singleApproverWorkflowDefinition,
+				externalReferenceCode: getRandomString(),
+			}
+		);
+
+	return workflowDefinition;
+}
```
Comment (larissacribeiro) [inline modules/test/playwright/tests/portal-workflow-kaleo-designer-web/utils/postSingleApproverCopy.ts:26]: hey @igor-franca, can you include a blank line at the end of the file?

**Review by larissacribeiro — CHANGES_REQUESTED** [review]
Hey @igor-franca, awesome work!! 🎉  there's just a tiny detail.

**Comment by igor-franca** [comment]
The playwright tests were successfully corrected and added to the workflow routine. 

Waiting to see if we can proceed with this change in routine.

File: modules/test/playwright/tests/portal-workflow-kaleo-designer-web/utils/postSingleApproverCopy.ts (hunk 1 above)
Comment (igor-franca) [reply modules/test/playwright/tests/portal-workflow-kaleo-designer-web/utils/postSingleApproverCopy.ts:26]: We already have this extra line when we view the file, I think by default git is not displaying it or vsCode is responsible for showing it to us, I don't know for sure.

But when trying to add one more extra line at the end of the file, it ends up being removed by sf, so I think it's ok 😄

Comment (larissacribeiro) [reply modules/test/playwright/tests/portal-workflow-kaleo-designer-web/utils/postSingleApproverCopy.ts:26]: nevermind this, Igor showed me that this last line is present at the file. For some reason git doesn't show it for any file, but all's good!

**Review by larissacribeiro — APPROVED** [review]
LGTM!! 🎉

**Comment by igor-franca** [comment]
ci:forward

**Pull request closed without merging**